app/views/users/new.html.erb

```diff
@@ -0,0 +1,50 @@
+
+<% 
+=begin %>
+# <form accept-charset="UTF-8" action="/users" method="post" >
+#   <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+#   <label for="username">Username:</label>
+#   <input type="text" id="username" name="user[username]"><br><br>
+#   <label for="email">Email:</label>
+#   <input type="text" id="email" name="user[email]"><br><br>
+#   <label for="password">Password:</label>
+#   <input type="password" id="password" name="user[password]"><br><br>
+#   <input type="submit" value="Submit">
+# </form>
+<% 
+=end %>
+
+<% 
+=begin %>
+# <%= form_tag(users_path, method: "post") do %>
+#   <%= label_tag(:username, "Name:") %>
+#   <%= text_field_tag(:username) %>
+#   <%= label_tag(:email, "Email:") %>
+#   <%= text_field_tag(:email) %>
+#   <%= label_tag(:password, "Password:") %>
+#   <%= text_field_tag(:password) %>
```

Comment:
- [ ] Here, kindly use the `password_field_tag` instead of `text_field_tag`.

Reply:
Thanks. `text_field_tag` changed to `password_field_tag`